File: Functions/2.py
def sum_upto_n(n):
    sum = 0
    for i in range(n+1):
        sum+=i
    return sum

def product_upto_n(n):
    p = 1
    for i in range(1,n+1):
        p*=i
    return p

print(sum_upto_n(10))
print(sum_upto_n(100))
print(product_upto_n(5))
print(product_upto_n(10))

File: Functions/2.cpp
#include<bits/stdc++.h>
using namespace std;

int sum_upto_n(int n){
    int sum = 0;
    for(int i=0;i<n+1;i++){
        sum+=i;
    }
    return sum;
}

int product_upto_n(int n){
    int p = 1;
    for(int i=1;i<n+1;i++){
        p*=i;
    }
    return p;
}

int main() {
    cout << sum_upto_n(10) << endl;
    cout << sum_upto_n(100) << endl;

    cout << product_upto_n(5) << endl;
    cout << product_upto_n(10) << endl;
    return 0;
}